C01/project-metadata/C01-ProjectId934-metadata.json

```diff
@@ -69,12 +69,43 @@
       }
     },
     "type": "Feature",
+    "place_name": "Mai Ndombe, Democratic Republic of the Congo, Africa",
     "geometry": {
       "type": "Point",
       "coordinates": [
         18.044855,
         -1.958494
       ]
     }
-  }
+  },
+  "regen:previewPhoto": {
+    "@type": "schema:URL",
+    "@value": "https://regen-registry.s3.amazonaws.com/projects/mai-ndombe/preview-photo.jpg"
+  },
+  "regen:galleryPhotos": {
```

Comment:
Here's a preview. I picked some random photos from airtable. I asked @erikalogie to pick her favorites, then I can update.

Reply:
looks good, thanks @mhagel !